lib/dependency-manager-adapters/npm.js

```diff
@@ -58,6 +61,14 @@ module.exports = CoreObject.extend({
       let cleanupTasks = [rimraf(path.join(adapter.cwd, adapter.packageJSONBackupFileName)),
         rimraf(path.join(adapter.cwd, adapter.nodeModulesBackupLocation))];
 
+      if (fs.existsSync(path.join(this.cwd, this.yarnLockBackupFileName))) {
+        cleanupTasks.push(rimraf(path.join(adapter.cwd, adapter.yarnLockBackupFileName)));
+      }
+
+      if (fs.existsSync(path.join(this.cwd, this.NPMShrinkWrapBackupFileName))) {
+        cleanupTasks.push(rimraf(path.join(adapter.cwd, adapter.NPMShrinkWrapBackupFileName)));
```

Comment:
Same as above, I would think that we can always do the `rimraf`...

Reply:
@rwjblue rimraf will throw an error if the file does not exist.  We will need to check for its' existence before removing it.